### PR TITLE
Makes syndie soft suit and helmet have the same description

### PR DESCRIPTION
**Pull request by imsxz**
Does as the title says. Currently, the descriptions are both different in a minor way. This really bugged me, so I am making a PR to change it.

🆑 imsxz
tweak: Edits the syndicate soft suit helmet's description to match the suit's.
/🆑

### CODE REVIEW

File: code/modules/clothing/spacesuits/syndi.dm [hunk 1]
```diff
@@ -4,7 +4,8 @@
 	desc = "Top secret Spess Helmet."
 	icon_state = "syndicate"
 	item_state = "syndicate"
-	desc = "Has a tag: Totally not property of an enemy corporation, honest."
+	desc = "Has a tag on it: Totally not property of of a hostile corporation, honest!"
+	w_class = WEIGHT_CLASS_NORMAL
```
Comment (Birdtalon) [inline code/modules/clothing/spacesuits/syndi.dm]: You added a weight class that wasn't there previously?

Comment (Birdtalon) [inline code/modules/clothing/spacesuits/syndi.dm]: You added a weight class that wasn't there previously, this should be stated in your PR.

Comment (imsxz) [inline code/modules/clothing/spacesuits/syndi.dm]: It was there previously, I copy pastad the description and it jumped down to that line

Comment (imsxz) [inline code/modules/clothing/spacesuits/syndi.dm]: Nevermind I'm an idiot, I'll remove it.

File: code/modules/clothing/spacesuits/syndi.dm [hunk 2]
```diff
@@ -4,7 +4,7 @@
 	desc = "Top secret Spess Helmet."
 	icon_state = "syndicate"
 	item_state = "syndicate"
-	desc = "Has a tag: Totally not property of an enemy corporation, honest."
+	desc = "Has a tag on it: Totally not property of of a hostile corporation, honest!"
```
Comment (deep-fried-faceplant) [inline code/modules/clothing/spacesuits/syndi.dm]: "of of a hostile..."

Comment (imsxz) [inline code/modules/clothing/spacesuits/syndi.dm]: check the master branch, that was there for the softsuit description already, I (and everyone else) just didn't notice it..

**Pull request merged**